### PR TITLE
Add pgbouncer

### LINKED ISSUE #9: Add pgbouncer to deployment
To address issue described in https://github.com/thoth-station/user-api/issues/665 I have deployed pgbouncer to our deployments. I used https://github.com/fabric8-analytics/fabric8-analytics-pgbouncer available on quay at https://quay.io/repository/openshiftio/bayesian-coreapi-pgbouncer 

We should make pgbouncer part of this role so that we always deploy pgbouncer. Also, the thoth configmap should be adjusted so that it points to pgbouncer instead of directly to postgres.

### PR DESCRIPTION
**Pull request by fridex**
Fixes: #9

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -156,6 +156,130 @@
               name: sosedoff/pgweb:0.11.3
             name: latest
 
+  - name: "create pgbouncer deployment"
+    k8s:
+      state: present
+      definition:
+        apiVersion: v1
```
Comment (goern) [inline tasks/main.yml]: I think this is not the correct api version?!

File: tasks/main.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@
           name: pgweb
           namespace: "{{ namespace }}"
           labels:
-            app: thoth
+            app: thoth-postgres
```
Comment (goern) [inline tasks/main.yml]: shouldnt we keep all the things thoth needs within the thoth app?

File: tasks/main.yml (hunk 1 above)
Comment (fridex) [reply tasks/main.yml]: Correct, adjusted also for other objects.

File: tasks/main.yml (hunk 2 above)
Comment (fridex) [reply tasks/main.yml]: +1, adjusted

**Comment by harshad16** [comment]
LGTM

**Review by goern — APPROVED** [review]

**Pull request merged**